### PR TITLE
Momentjs

### LINKED ISSUE #431: Re-Enable momentjs Suffixes
Looking at the screen which lists multiple upcoming departures of a particular route at a stop, I saw this:

Real time | "Scheduled" | "Estimated"
--- | --- | ---
8:24 am | 8:24 am | 8:28 am (4 minutes)
8:28 am | 8:24 am (4 minutes) | 8:28 am

Since I didn't look at my phone during the intervening 4 minutes, it caught me off guard a little. Perhaps we should distinguish between "in 4 minutes" and "4 minutes ago"?

### PR DESCRIPTION
**Pull request by artarazavi**
closes #431

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,8 +46,11 @@
     "ng2-pagination": "^2.0.1"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "2.1.3",
-    "@types/jasmine": "^2.5.41",
+    "@ionic/app-scripts": "^1.3.7",
```
Comment (werebus) [inline package.json]: It looks like you downgraded a bit here.  You should update your copy of Ionic to the [recommended version here](https://github.com/umts/pvta-multiplatform#prerequisites).

File: src/pages/stop/stop.component.ts [hunk 2]
```diff
@@ -184,8 +184,8 @@ export class StopComponent {
       sExact: moment(sdt).format('LT'),
       eExact: moment(edt).format('LT'),
       // ex: '6 minutes'
-      sRelativeNoPrefix: moment(sdt).fromNow(true),
-      eRelativeNoPrefix: moment(edt).fromNow(true)
+      sRelativeNoPrefix: moment(sdt).fromNow(),
```
Comment (werebus) [inline src/pages/stop/stop.component.ts]: Minor nit-pick: the `NoPrefix`, I believe, refers to the fact that there _isn't_ an 'in' or an 'ago'.

**Review by werebus — DISMISSED** [review]

File: src/pages/stop/stop.component.ts (hunk 2 above)
Comment (akaplo) [reply src/pages/stop/stop.component.ts]: Yup, just the variable naming should be changed.

**Review by akaplo — DISMISSED** [review]
Change the variable name that @werebus and I commented on, and possibly just `git checkout` `package.json` from `master` - you shouldn't need to make any changes to it.  But great job! Happy to see your contribution 🎉

File: package.json (hunk 1 above)
Comment (akaplo) [reply package.json]: The comment I left on #434 applies here verbatim.

**Review by werebus — COMMENTED** [review]
Looks good. Although, don't you have to change the name in [the page](https://github.com/umts/pvta-multiplatform/blob/momentjs/src/pages/stop/stop.html) too?

**Comment by mboneil10** [comment]
@werebus yup

**Review by werebus — APPROVED** [review]

**Comment by mboneil10** [comment]
ugh what did i do

**Comment by mboneil10** [comment]
OH, duh. Obviously have to make changes to `moment`.

**Comment by mboneil10** [comment]
uhhhh ok I'll just merge

**Pull request merged**